action/tests/test_main.py

```diff
@@ -98,24 +102,12 @@ def test_creates_virtualenv(tmp_path, main_patch):
         ],
     ),
     dict(
-        run_main_env={"INPUT_LINT": "pylint"},
-        expect=["darker[color,isort]", "pylint"],
-    ),
-    dict(
-        run_main_env={"INPUT_LINT": "pylint,flake8"},
-        expect=["darker[color,isort]", "pylint", "flake8"],
-    ),
-    dict(
-        run_main_env={"INPUT_LINT": "  flake8  "},
-        expect=["darker[color,isort]", "flake8"],
-    ),
-    dict(
-        run_main_env={"INPUT_LINT": "  flake8  ,  pylint  "},
-        expect=["darker[color,isort]", "flake8", "pylint"],
+        run_main_env={"INPUT_LINT": "dummy"},
+        expect=["darker[color,isort]"],
     ),
     dict(
-        run_main_env={"INPUT_LINT": "  flake8  >=  3.9.2  ,  pylint  ==  2.13.1  "},
-        expect=["darker[color,isort]", "flake8>=3.9.2", "pylint==2.13.1"],
+        run_main_env={"INPUT_LINT": "dummy,foobar"},
+        expect=["darker[color,isort]"],
```

Comment:
Question: is the point of these dummy inputs to test that using `--lint` flag doesn't break anything?